### PR TITLE
Update template

### LINKED ISSUE #4: Template generates project using deprecated JSApp trait
the JSApp trait is deprecated. See the ScalaDoc of JSApp for alternatives which do not require applications to depend on JSApp.

### PR DESCRIPTION
**Pull request by fdietze**
fixes #4

### CODE REVIEW

**Comment by fdietze** [comment]
Should I merge? I think that a lot of newcomers use the original very outdated template.

File: src/main/g8/src/test/scala/$name__camel$/JSDomSpec.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package $name;format="camel"$
+
+import org.scalatest._
+import org.scalajs.dom._
+import monix.execution.ExecutionModel.SynchronousExecution
+import monix.execution.schedulers.TrampolineScheduler
+import monix.execution.{Cancelable, Scheduler}
+
+abstract class JSDomSpec extends FlatSpec with Matchers with BeforeAndAfterEach {
```
Comment (britter) [inline src/main/g8/src/test/scala/$name__camel$/JSDomSpec.scala]: Do you really want to force `FlatSpec` and `Matchers` on your users?

File: src/main/g8/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+target/
+.idea/
```
Comment (britter) [inline src/main/g8/.gitignore]: The .idea folder should be excluded by a user's global gitignore file. I recommend to only put build tool related entires into the project's gitignore.

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.17
```
Comment (britter) [inline project/build.properties]: Why not 1.x?

File: src/main/g8/src/test/scala/$name__camel$/JSDomSpec.scala (hunk 1 above)
Comment (fdietze) [reply src/main/g8/src/test/scala/$name__camel$/JSDomSpec.scala]: Are there any disadvantages with this approach? Feel free to simplify/improve it with a PR.

File: src/main/g8/.gitignore (hunk 2 above)
Comment (fdietze) [reply src/main/g8/.gitignore]: Agreed, I do the same with my own projects. But the goal here is to lower the barrier for newcomers.

File: project/build.properties (hunk 3 above)
Comment (fdietze) [reply project/build.properties]: At that time sbt 1.0 had file watching issues on macs, I think.
But they should be fixed with the latest sbt 1.0.6.

**Pull request merged**